### ISSUE TITLE
Bitcoin Bot no longer showing ticker price

### ISSUE THREAD

**Issue by GodofWar9000** [muhammadmuzzammil1998/BitcoinBot#21]
![image](https://user-images.githubusercontent.com/9539678/43781566-31f5713a-9a2b-11e8-92ee-b15456cf29ed.png)

See above picture – as it doesn't show the price.

Also, can we have a bitmex and bitfinex price option for USD in the ticker? Bitmex's volume is the largest, and then bitfinex's is second. Bitmex BTC/USD should probably be the ticker price as it has the largest effect on BTC-USD price.

**Comment by muhammadmuzzammil1998** [comment]
Thanks for bringing this to my attention. I will look into it as soon as possible.

**Comment by muhammadmuzzammil1998** [comment]
Hi! 
So, I just checked and it seems to be working fine. Can you please check if the problem is still there?
![image](https://user-images.githubusercontent.com/12321712/44014578-4b12f958-9eea-11e8-9c7b-70fa9c2bc718.png)

As for the price options for the ticker, I have already discussed it in issue #19,

> [...] If a user asks for USD per BTC they are not gonna get the same price as the status and that's bad. Furthermore, I would have to manage two APIs which I can but it would increase the delay in response.

---
---

### Also, do you have any comments for issue #19?

**Comment by mastermood** [comment]
![image](https://user-images.githubusercontent.com/9539678/44064293-3f19ad8e-9f32-11e8-9b32-e40f1b29c6f3.png)
The bot will show the ticker price a few time sin a day but is not working 90% of the time. 

Also, I think bitmex-USD prices should be used given you mentioned two APIs would be redundant. 

![image](https://user-images.githubusercontent.com/9539678/44064313-5e13ce18-9f32-11e8-81a8-715da4a99275.png)
Note that Bitmex has the highest volume & also influences the price of BTC throughout the market.

**Comment by muhammadmuzzammil1998** [comment]
Yes, I noticed it after a while that the status appeared for a few seconds and then disappeared. I don't think it is a problem in my code because it was working just fine. 
I think it is because of either discord or coinbase API. I highly doubt the latter as it can't affect bot's status.

Anyway, I'll look into it. [https://discord.gg/D9JpAUh](https://discord.gg/D9JpAUh) here is a link to the development server I just created. Would be nice to have someone there 😂
